x/concentrated-liquidity/lp.go

```diff
@@ -8,7 +8,10 @@ import (
 	types "github.com/osmosis-labs/osmosis/v12/x/concentrated-liquidity/types"
 )
 
-func (k Keeper) Mint(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, liquidityIn sdk.Dec, lowerTick, upperTick int64) (amtDenom0, amtDenom1 sdk.Int, err error) {
+func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, amount0Desired, amount1Desired, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64) (amtDenom0, amtDenom1 sdk.Int, err error) {
+	// TODO: calculate from amounts given
+	liquidityIn := sdk.MustNewDecFromStr("1517.882323")
```

Comment:
why are we using hard coded values?

Reply:
Synched offline

Reply:
To make tests pass. This value has to be calculated from amounts given